src/pint/models/binary_ddk.py

```diff
@@ -99,3 +99,8 @@ def validate(self):
                 raise MissingParameter(
                     "DDK", "DDK model needs proper motion parameters."
                 )
+        if self.SINI.quantity is not None:
+            raise AttributeError(
```

Comment:
This seems like a ValueError - the user has passed a bad `.par` file or tried to create an invalid model. More, because TimingModel objects use both properties and `__getattr__`, AttributeErrors behave strangely.

Reply:
Changed to ValueError.